Sources/Sentry/SentryOptions.m

```diff
@@ -23,6 +23,17 @@ - (_Nullable instancetype)initWithDict:(NSDictionary<NSString *, id> *)options
         if (nil != error && nil != *error) {
             return nil;
         }
+
+        // If no user-defined release, use default.
+        if (nil == self.releaseName) {
+            NSDictionary *infoDict = [[NSBundle mainBundle] infoDictionary];
+            if (nil != infoDict) {
+                self.releaseName = [NSString stringWithFormat:@"%@@%@+%@",
+                                    infoDict[@"CFBundleIdentifier"],
+                                    infoDict[@"CFBundleShortVersionString"],
+                                    infoDict[@"CFBundleVersion"]];
+            }
+        }
```

Comment:
Would it make sense to also set the default if the user calls `[options setReleaseName:nil];`? What do you think @marandaneto and @HazAT ?

Reply:
mmm not sure if it's a good idea, if the user calls passing `null`, its purposely or a mistake, I'd rather make the method non-null then